README.md

```diff
@@ -1,3 +1,21 @@
 # 3chLine
 
 a [Sails](http://sailsjs.org) application
```

Comment:
we talk about our app name and rename `3chLine` -> `3chPiazza`
so i'll remove l.3 and modify app name :+1: